modules/components/pages/redpanda/about.adoc

```diff
@@ -2,16 +2,43 @@
 
 component_type_dropdown::[]
 
-As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs to a topic in a Redpanda cluster.
+// tag::single-source[]
 
-The configuration for this feature lives under the `redpanda` namespace, with the following default values:
+// © 2024 Redpanda Data Inc.
 
-[source,yaml]
-----
+As well as the xref:components:logger/about.adoc[default logger], you can configure Redpanda Connect to send logs and status updates to topics on a Redpanda cluster.
+
+The `redpanda` namespace contains the configuration of the logging server, which uses the following default values:
```

Comment:
I've called this the "logging server" but is that correct? Was updated from "this server".

Reply:
It's not a server, it's just a feature that enables observability data to be sent to redpanda topics.

Reply:
@Jeffail. I've updated the text so that to call it a service, unless there is a better word to describe this.

Reply:
Yeah that's fine with me